### ISSUE TITLE
About the training time

### ISSUE THREAD

**Issue by manainszu** [arthurhero/Long-LRM#8]
Hey , bro, I have one issue during training the example dataset, I set the train_steps 1000. But one iteration will cost 10 sec on Nvidia-L20. Whether I have same wrong setting or options?

This is my config:

checkpoint_dir: '/mnt/Long-LRM/checkpoints'
evaluation_dir: "evaluation"
api_key_path: "./api_keys.yaml"
use_tf32: true
use_amp: true
amp_dtype: "bf16"
data:
  data_path: "/mnt/Long-LRM/data/example_data/mydesk.txt"
  resize_h: 256
  square_crop: true
  input_frame_select_type: "uniform"
  target_frame_select_type: "random"
  num_input_frames: 32
  num_target_frames: 8
  target_has_input: true
  min_frame_dist: 64
  max_frame_dist: 128
  shuffle_input_prob: 0.5
  reverse_input_prob: 0.5
model:
  num_layers: 24
  patch_size: 8
  dim: [256, 1024]
  block_type: "mmmmmmmt"
  merge_layers: [8]
  transformer:
    head_dim: 64
  mamba2:
    d_state: 256

  num_global_tokens: 2
  gaussians:
    sh_degree: 3
    near_plane: 0.01
    far_plane: 1000000.0
    scale_bias: -6.9
    scale_max: -1.2
    opacity_bias: -2.0
    align_to_pixel: true
    max_dist: 500.0
data_eval:
  target_frame_select_type: "uniform"
  target_has_input: false
  min_frame_dist: 96
  max_frame_dist: 96
  shuffle_input_prob: 0.0
  reverse_input_prob: 0.0

training:
  lr: 0.00004
  beta1: 0.9
  beta2: 0.95
  weight_decay: 0.05
  train_steps: 1000
  batch_size_per_gpu: 1
  grad_accum_steps: 4
  warmup_steps: 2000
  scheduler_type: "cosine"
  grad_clip_norm: 10.0
 # dataloader
  num_workers: 4
  prefetch_factor: 32

  # wandb
  wandb_offline: true
  wandb_entity: <YOUR_WANDB_ENTITY>
  wandb_project: <YOUR_WANDB_PROJECT>

  # losses
  l2_loss: 1.0
  perceptual_loss: 1.0
  # perceptual loss settings from https://github.com/zhengqili/Crowdsampling-the-Plenoptic-Function/blob/master/models/networks.py#L1478
  perceptual_out_idx: [4, 9, 14, 23, 32]
  perceptual_out_weights: [0.3846, 0.2083, 0.2703, 0.1786, 6.6667]
  perceptual_feature_scale: 255.0
  # weights from https://www.vlfeat.org/matconvnet/pretrained/
  perceptual_vgg_weights: "/mnt/tongbingkai/Long-LRM/model/matconvnet_vgg19.pt"
  opacity_loss: 0.1
  gaussian_depth_loss: 0.01

  # logging
  print_every: 20
  wandb_every: 50
  checkpoint_every: 500
  vis_every: 250
  save_gaussian_every: 1000
  save_video_every: 250
